BrainPortal/app/models/signup.rb

```diff
@@ -72,15 +72,15 @@ def user_full #:nodoc:
   alias full_name full
 
   def approved? #:nodoc:
-    self.approved_by.present? && self.approved_at.present?
+    self.user_id.present?
```

Comment:
This will make all previous records already approved seem to no longer be approved! PLease return the check to what it was.